tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1113,13 +1118,35 @@ def _run_pseudolabeling(self, unlabeled_data: pd.DataFrame, max_iter: int,
                 X_test = X_test.loc[test_pseudo_idxes[~test_pseudo_idxes].index]
                 previous_score = current_score
 
+        # Should be done at the end. When done at each iteration caused the model to get much worse
+        if fit_ensemble:
+            logger.log(15, 'Fitting weighted ensemble using models trained with pseudo labeled data')
```

Comment:
isn't this a vacuous statement (ie pseudo-models are always used in ensemble to my understanding)?
Shouldn't it instead say something like:
"Creating weighted ensemble of models trained with pseudo labeled data and models trained on original data"